### ISSUE TITLE
Embed Google or Typekit fonts?

### ISSUE THREAD

**Issue by befreestudios** [bauhausjs/phantom-html2pdf#24]
Anyone know of a way to embed google web fonts, or typekit fonts for use in a PDF ? I'm assuming you might be able to do this by way of the runnings file, but I'm at a loss. Anyone tried this before, or have any suggestions?

Thanks in advance...

**Comment by dustin-H** [comment]
Have you tried to just add the font link to your HTML file like this?:

```html
<link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
```

And use it in CSS:

```css
font-family: 'Open Sans', sans-serif;
```

**Comment by befreestudios** [comment]
@dustin-H Thanks, but that doesn't work. Actually when I do that it generates a blank pdf

**Comment by befreestudios** [comment]
FYI... for anyone interested, came across this article. We are going to try to implement this on our node server and see how it works out...

https://medium.com/@stockholmux/besting-phantomjs-font-problems-ee22795f5c0b#.rpx1cf2zi

**Comment by befreestudios** [comment]
@dustin-H 

In case anyone is interested (and it may be worth documenting), the above solution I posted worked. Essentially we needed to have the font we wanted rendered in the pdf on the production node server server that was running the pdf-generation (the link explains in detail)...

**TLDR;**

Custom fonts on PhantomJS step-by-step for Ubuntu
- Get a Type1 version of the font you need. I needed to do a conversion from TrueType— there are lots of tools to do this with.
- Upload your Type1 font to your ‘/usr/share/fonts/type1' directory.
- Run ‘fc-cache -fv’
- Enjoy having PhantomJS with your new font.

**Comment by dustin-H** [comment]
Hi @befreestudios ,

thanks a lot for that information. I will add them to the docs soon!
